views/home.erb

```diff
@@ -5,6 +5,15 @@
 <h1 class="dcf-txt-h3">Welcome to Innovation Studio Manager</h1>
 
 <h5>Hello, <%= @user.full_name %> (<%= @user.username %>)</h5>
+<% if @user.expiration_date > Date.today%>
```

Comment:
Don't hate me please this is very minor but can we get a space separating Date.Today and %>

Reply:
Done.